src/ore/src/antichain.rs

```diff
@@ -78,14 +78,6 @@ impl<T: PartialOrder + Ord + Clone + std::fmt::Debug> AntichainToken<T> {
         let mut new_frontier = Antichain::new();
         new_frontier.extend(frontier.into_iter());
 
-        if !<_ as PartialOrder>::less_equal(&self.antichain.frontier(), &new_frontier.borrow()) {
```

Comment:
As discussed offline, maybe better is to just move this file to the coord package.